dist-persist/wbstack/src/Shim/Web.php

```diff
@@ -2,11 +2,13 @@
 
 require_once __DIR__ . '/../loadShim.php';
 
-// Try and get the wiki info, for fail with a 404 web page
-if(!\WBStack\Info\GlobalSet::forDomain( $_SERVER['SERVER_NAME'] ) ) {
-    http_response_code(404);
+// Try and get the wiki info or fail
+try {
+    \WBStack\Info\GlobalSet::forDomain($_SERVER['SERVER_NAME']);
+} catch (Exception $ex) {
+    http_response_code($ex->getCode());
     echo "You have requested the domain: " . $_SERVER['SERVER_NAME'] . ". But that wiki can not currently be loaded.\n";
```

Comment:
We could also switch on the error code here and provide different messages. Not sure if it's needed?